### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by theuranus**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  "Other",
+];
+
+function main() {
+  const ajv = new Ajv();
+
+  const schema = {
+    type: "object",
+    patternProperties: {
+      "^[\\w\\d]{56}$": {
```
Comment (longngn) [inline index.js:26]: [0-9a-f]

File: index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          },
+          socialLinks: {
+            type: "object",
+            properties: {
+              website: { type: "string" },
+              discord: { type: "string" },
+              telegram: { type: "string" },
+              twitter: { type: "string" },
+              coinMarketCap: { type: "string" },
+              coinGecko: { type: "string" },
```
Comment (longngn) [inline index.js:47]: format url

Comment (hieupnh) [inline index.js:47]: The type of properties should be `uri` instead of `string`

Comment (hieupnh) [inline index.js:47]: The type of properties should be `uri` instead of `string`

**Review by hieupnh — APPROVED** [review]

**Pull request merged**